docs/src/main/sphinx/release/release-377.md

```diff
@@ -0,0 +1,29 @@
+# Release 377 (13 Apr 2022)
+
+## General
+
+* Add support for ANSI compliant `trim` function. ({issue}`11236`)
+* Fix bug when in certain rare cases fault-tolerant execution of queries
```

Comment:
> Fix incorrect results when queries execute in fault-tolerant mode.

Reply:
haha .. yeah .. I was uncomfortable with that wording too ..